src/System.IO/tests/TextWriter/TextWriterTests.netcoreapp.cs

```diff
@@ -52,5 +54,62 @@ public void WriteLineCharSpanTest()
                 Assert.Equal(new string(rs.Span) + tw.NewLine, tw.Text);
             }
         }
+
```

Comment:
Could you please add tests for (1) string builders with multiple chunks (2) empty string builders.

Reply:
Done

Reply:
Which test creates a StringBuilder with multiple chunks?  I'm probably just missing it, but they all seem to construct a StringBuilder from a string, which I thought would just create a single chunk.